addon/components/item-picker/template.hbs

```diff
@@ -53,6 +53,7 @@
     {{#if selectMultiple}}
       <div class="item-picker-status">
         {{t (concat _i18nScope "selectedCount") count=itemsToAdd.length}}
+        <a {{action 'deselectAll'}} class="anchor-tag-button">{{t (concat _i18nScope "deselectAll")}}</a>
```

Comment:
I don't think we need to use an anchor tag with custom CSS here. Why not just a `button` with `class="btn btn-link"`?

Reply:
The original designs called for a link ([ticket](https://esriarlington.tpondemand.com/entity/83372)). I talked to @FelFly and @samanthahunter and I think the ultimate decision was to keep it as a link and have it in a stationary position unless it's at the end of a pill set (like when they will be applied to the search)

@FelFly @samanthahunter Is that correct?

Reply:
But here's the thing.
- This isn't a "link" - semantically a button is more appropriate.
- By (inappropriately) using a link for this, you need to add custom css.
- But by using the more semantically appropriate `button` and applying the `btn-link` class, it will look like a link.

The designs prescribe how it should look and behave but they don't prescribe _how_ it should be built.

Reply:
Makes sense. I can make it a button and put `btn-link` on it 👍

Reply:
You can make this into a button. Mike's correct. There's a bootstrap class `.btn-link` that will make it appear visually like a link. Buttons perform actions, links (usually) open pages. 

And as an aside, a `<a>` element with no href doesn't receive keyboard focus very well, so there's more jumping through hoops to do it that method - we had very inconsistent user-dropdown menus for a while and the `<a>` link rather than `<button>` was part of the problem.